src/ui/web/src/composable/UIClient.ts

```diff
@@ -16,6 +16,7 @@ export default class UIClient {
       reject: (reason?: any) => void;
     }
   >;
+  private _toBeSent: Array<string>;
```

Comment:
It's an interactive UI. No buffering of any kind is needed, the user shall be notified directly. 
The buffer hides the error here, stack user actions doing UI request without rate control, etc. 
Keep it simple and stupid.